assets/src/blocks/Columns/ColumnsTasks.js

```diff
@@ -15,47 +15,47 @@ export const ColumnsTasks = ({ isCampaign, columns, no_of_columns }) => (
           const hasImage = attachment !== 0 && attachment !== undefined;
 
           return (
-            <div key={`column-${index}`} className='col-md-6 col-lg column-wrap step-info'>
-              <span className='step-number'>
+            <div key={`column-${index}`} className="col-md-6 col-lg column-wrap step-info">
+              <span className="step-number">
                 <span
-                  className='step-number-inner'
-                  data-ga-category='Columns Block'
-                  data-ga-action='Task Number'
-                  data-ga-label='n/a'
+                  className="step-number-inner"
+                  data-ga-category="Columns Block"
+                  data-ga-action="Task Number"
+                  data-ga-label="n/a"
                 >
                   {index + 1}
                 </span>
               </span>
               {title &&
                 <h5>
-                  {cta_link ?
-                    <a
+                  {cta_link
+                    ? <a
                       href={cta_link}
                       target={link_new_tab ? '_blank' : ''}
-                      data-ga-category='Columns Block'
-                      data-ga-action='Title'
-                      data-ga-label={cta_link}
+                      data-ga-category="Columns Block"
+                      data-ga-action="Title"
+                      data-ga-label={cta_link} rel="noreferrer"
                     >
                       {title}
-                    </a> :
-                    title
+                    </a>
+                    : title
                   }
                 </h5>
               }
               {description &&
-                <p dangerouslySetInnerHTML={{ __html: description }} />
+                <p dangerouslySetInnerHTML={{__html: description}} />
               }
               {hasImage &&
-                <img src={attachment} alt='' loading='lazy' />
+                <img src={attachment} alt="" loading="lazy" />
```

Comment:
Thinking about the A11Y, we would need to consider to a add an `eslint` rule (if it exists) to validate empty `alt`. This is a very important attribute to respect.

Reply:
That is true, but also thinking about the [documentation](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/docs/rules/alt-text.md), they state that if the image is used as an icon, button or control element, the `alt` prop should be an empty string. So we could look into that.

Reply:
Here is another interesting explanation of when it's recommended (or not necessary) to set an empty string.

According to the [A11y project](https://www.a11yproject.com/posts/alt-text/)
> Use an empty alt attribute for any image that is decorative or not necessary for understanding the content of the page. For example, an image providing decoration to an article's heading could be hidden by giving the image an alt="" or simply alt